Description: Fixes the MANIFEST.in to install all what's missing
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2013-09-26

--- ceilometer-2013.2~b3.orig/MANIFEST.in
+++ ceilometer-2013.2~b3/MANIFEST.in
@@ -11,4 +11,4 @@ recursive-include nova_tests *.py
 global-exclude *.pyc
 recursive-include public *
 recursive-include ceilometer/locale *
-recursive-include ceilometer *.html
+recursive-include ceilometer *.html *.css *.png *.js *.ini *.py
